test/unit-test/main.tf

```diff
@@ -0,0 +1,229 @@
+module "ecs" {
+
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-ecs?ref=61ed2af4ba963647931ed0359939787a0f0d74f7"
```

Comment:
shouldn't this refer to the local root directory, so the code being tested is always to most recent version?

Reply:
Will update with a tagged version once these changes are merged